stable/rethinkdb/templates/rethinkdb-secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rethinkdb-password: {{ .Values.rethinkdbPassword | b64enc | quote }}
```

Comment:
@thomasmodeneis Whilst a serious admin will take care to generate a secure password and protect the cluster, it's always safer to set secure defaults for an installation. [MongoDB](https://www.theregister.co.uk/2017/01/09/mongodb/) is a great example of where not using secure defaults can go horribly wrong.

The `helm upgrade` problem is an issue, but something we need to push for fixing upstream in Helm, until then we should document the upgrade procedure and explain that the password needs to be set when upgrading.

If we feel strongly about not randomly generating a password because of this issue, we definitely need to set the default service type to ClusterIP (however I think doing both is best).